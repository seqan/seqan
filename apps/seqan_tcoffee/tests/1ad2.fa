>rk1_cyapa
EEQLYEPLEAIQLLKQISTAKFIETVEVHCRLGIDPKYNDQQLRATVTKAGKLEFRNDRTGIVHVLFGKA
NFSEEALLGNLKAVQEAVDRNRPSGVKGKYWKSIYITSTMSPSIEVDLPNGTGQTLRVAVITRGEKINEA
STAGADLVGAEELIDEIQNGRLDFDCLIATPDMMPQVAKLGRVLGPRGLMPSPKGGTVTFDLAQAINDF
>rl1_halvo
VDAVSRALDEAPGRNFRETVDLAVNLRDLDLNDPSKRVDESIVADETDFFVAEAGLMQDIGRYLGTVLGP
RGKMPTPLQPDDDVVETVNRMKNTVQLRSRDRRTFHTRVGADDMTPDEIAENIDVIVRRLEATLEKGPLN
IDSVYVKTTMGPSVEVLPSGTGQDTQIVVFATGETAAEDAADEVLGPDELEDFGDDTDAAKDL
>rl1_metja
VKEARELAKPRNFTQSFEFIATLKEIDMRKPENRIKTEVVAKAHDFFIAQADLMPLIGRYMGVILGPRGK
MPKPVPANANIKPLVERLKKTVVINTRDKPYFQVLVGNEKMTDEQIVDNIEAVLNVVAKKYEKGLLPHGR
GKEAKIAVIGTGDLAKQAEELGLTVIRKEEIEELGKNKRKLRKIYHIKDAYVKLTMGPAVKVK
>rl1_sulso
AIQNEWFLINQESMALAGRILGPALGPRGKFPTPLPNTADISEYINRFDKESLIEALKLALSTEYNVKRN
FTQSVEIILTFKGIDMKKGDLKLREIVPKRSVIVKTKDQPQVQVFIGTEDMKPEDLAENAIAVLNAIENK
AKVELPKQPSKAKRVLVVPSSEQLEYAKKASPKVVITREELQKLQGQKRPVKKLTNLRNIYVKTTMGKAV
KVK
