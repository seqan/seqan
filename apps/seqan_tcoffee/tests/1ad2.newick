(rk1_cyapa,(rl1_halvo,(rl1_metja,rl1_sulso)))
