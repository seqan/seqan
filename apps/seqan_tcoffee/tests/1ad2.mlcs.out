>rk1_cyapa
----EEQ-----------------LY-E---PL-----E-------AIQ--L----LKQI
---------------STA-------KF------------I-------E----T-------
---V--EVHCRL-----GI----DPK--------YNDQQLRA--------T--VTKA---
-----G----------K-----L-----------------EF------R-------N---
---------DR-TGIV------H---VLF-----GKANFSE-------------E----A
LLG-NLKA--VQEA-----VDRNR-------------------P----SG----VKGKYW
---KSIYITSTMS----PSI--EVD---L-PN--------GTGQ-TLR----VAVI--TR
G-E-KINEAS----TAGA--DLVGA----EE-LI------DEIQNGRL-DF-DCLIATPD
MMPQVA--K-----LG----RV----LGP---RGLMPSPKGGTVTF-------DLAQAIN
---DF------------
>rl1_halvo
VD-----AV-----------SR-AL-DE-A-P-GRNFRETVD-L--A--VNL--RDL---
D------L-N--DP-S---------K-----RV-D-ES-IVAD----E----T-------
DFFV-AE---------AG--LMQD----IGR---Y----L---G------T--V------
-L---GPR-----G--KMPTP-LQPDDDV---------V-E-T-VN--RM-----KN-TV
QLRS----RDRRT--------FHTR-V-------G-A----DD---MTP-D---EI---A
---EN---IDV------I-V-R-RL---E--ATL---E--KG-PLNIDS-----V---Y-
-V-K----T-TM-G---PS-V-EV----L-P--------SGTGQDT--QI--V-V-FAT-
G------E------TA-A-ED---A--AD-EVL-----GPDE-----LEDFGD------D
--------------------------------------------T--------D-A-A--
--KD-L-----------
>rl1_metja
--VK-E-A-------------R-----ELAKP--RNF--T------------------Q-
---------------S--FEFIATLK----------E--I--DMRKPENRIKTEVVAKAH
DFF-IA------Q---A--DLM----PLIGR---Y-------MG---------V---I--
-L---GPR-----G--KM--PK--P---VPANANIKPLV-E-------R-L----K-KTV
----VINTRD------K--PYF---QV--L-V--G--------NEKMT--D---E-Q---
--------I-V---DN-I----------E--A--V---------LN--------V-----
-VAK----------KY------E--KG-LLP-H-------G------------------R
GKEAKI--A-VIG-T-G---DL--AKQA-EE-L------------G-L---------T--
----V-IRKEEIEELGKN-KR---K-L-----R-------------KIYH-IKD---A--
YVK--LT-MGP-AVKVK
>rl1_sulso
-------A-IQNEWFLINQES-MAL----A---G---R----ILGPA----LGPR-----
-GKFPTPLPNTAD-IS-----------EYINR-FDKESLI-------E------------
-----A-------LKLA---L----------STEY---------NVKRNFTQSV----EI
ILTFKG--IDMKKGDLK-----L----------------RE--IV-PKR--SVIVK--T-
----------------KDQP-----QV---QVFIG------------T-EDMKPE--DLA
---EN-----------AIAV----LNAIENKA---KVELPK-QP----S-KAKRV-----
LV--------------VPS--SE----QL----EYAKKAS----------PKV-V-----
-------------IT---RE----------E-L-QKLQG---------------------
------------------QKR-PVKKL--TNLR-----------------NI--------
YVK---TTMG-KAVKVK
